agent/lib/kontena/workers/service_pod_manager.rb

```diff
@@ -31,7 +31,8 @@ def start
 
       subscribe('service_pod:update', :on_update_notify)
       subscribe('service_pod:event', :on_pod_event)
-      every(30) do
```

Comment:
Actually we should probably do similar loop here as in volume manager.. or change volume manager to do `every`. /cc @jnummelin